docs/plugins/verifier/notation.md

```diff
@@ -88,7 +88,7 @@ Note: CLI is NOT SUPPORTED.
 
 > NOTE 2: `verificationCertStores` supersedes `verificationCerts` if both fields are specified.
 
-> NOTE 3: `verificationCertStores` currently supported values for `trust-store-type` are `ca`, `signingAuthority` and `tsa`(coming soon).  This change is backward compatible, the implementation supports both original CRs, which contain no trust store type, and CRs specifing `trust-store-type` for each certificate collection. In the sample above, `ca:certs` is the trust store specifing and the `certs` suffix corresponds to the `certs` certificate collection listed in the `verificationCertStores` section. 
+> NOTE 3: `verificationCertStores` currently supported values for `trust-store-type` are `ca`, `signingAuthority` and `tsa`(coming soon).  This change is backward compatible, the implementation supports both original CRs, which contain no trust store type, and CRs specifing `trust-store-type` for each certificate collection. Time-stamping does not support original CRs, which contain no trust store type. In the sample above, `ca:certs` is the trust store specifing and the `certs` suffix corresponds to the `certs` certificate collection listed in the `verificationCertStores` section. 
```

Comment:
When referring to `original CR`, it sounds a bit confusing to me. I feel we don't need to mention original/new CRs, but clarify the behavior change. e.g.
For backward compatibility, users can either specify the truststore type or omit it, in which case it will default to the ca type.
And probably you can provide 2 concise examples to be more intuitive.

Reply:
Sure. I would go ahead and provide 2 examples.